src/libcore/macros.rs

```diff
@@ -393,30 +388,10 @@ macro_rules! write {
 
 /// Write formatted data into a buffer, with a newline appended.
 ///
-/// On all platforms, the newline is the LINE FEED character (`\n`/`U+000A`) alone
-/// (no additional CARRIAGE RETURN (`\r`/`U+000D`).
```

Comment:
oops, I meant to move that into write, but it doesn't make sense there and I guess I forgot to put it back after not including it in write. Thanks!